### ISSUE TITLE
Missing labels in search results checkboxes

### ISSUE THREAD

**Issue by taofeeqhamzat** [mozilla/perfcompare#302]
The checkboxes in the SearchResultsListItem components are missing labels and this causes accessibility tests to fail

**Comment by amyra98** [comment]
Can you please assign me this issue @taofeeqhamzat

**Comment by taofeeqhamzat** [comment]
Hi @amyra98, I don't have the authority to assign issues in this project. This issue is was created in relation to a fix I've made for some accessibility issues failing in the test. 
This is a work in progress, there are more accessibility issues left to be fixed. Perhaps we could still work together on those.

Also, @kimberlythegeek mentioned earlier that we can freely work on issues without waiting for them to be assigned.

**Comment by amyra98** [comment]
okay cool can you tell me how to help you and what resources should I look for.. @taofeeqhamzat

**Comment by taofeeqhamzat** [comment]
If you've read [this contribution guide](https://docs.google.com/document/d/1NLKAR5B2Rc80lRUYWzU2ISrQCQhR4VsW5Qkz1hxE5n8), you'll know that we can find the accessibility test results [here](https://app.circleci.com/pipelines/github/mozilla/perfcompare/1103/workflows/8aaaeeb7-a7d5-422a-90fa-29257fe7c1e4/jobs/1107/parallel-runs/0/steps/0-107), we can start there. 
> Alternatively, you could run the test on your machine with `npm run test:a11y || true`.

So far, it looks like two components (or components rendered by them), `SearchResultsList` and `SelectedRevisionsTable`,  are failing the tests for several reasons. This issue addresses just one set of those reasons in one of the components and the accessibility test is expected to be updated to cover more components --It's all work in progress.

MaterialUI is the component library used by the project, so we could keep exploring the [MUI docs](https://mui.com/material-ui/getting-started/overview/) to figure out how to implement the components in ways that they pass the tests. I imagine you could work on one while I work on the other component. Two heads.
If it's okay, I'd like to find and message you from the group channel to discuss further.

@amyra98

**Comment by taofeeqhamzat** [comment]
I have updated this issue to cover all the accessibility issues in 'SearchResultsList'

**Comment by oburasamuel** [comment]
@taofeeqhamzat @amyra98 Ok lets collaborate in this!

**Comment by taofeeqhamzat** [comment]
@oburasamuel check out #305. What do you think? I did some work, but we could do more.
Please share your ideas.

**Comment by oburasamuel** [comment]
This was nice #305. I think you are in the right course.
